src/openrct2-ui/windows/TitleEditor.cpp

```diff
@@ -338,41 +342,47 @@ static void window_title_editor_mouseup(rct_window *w, rct_widgetindex widgetInd
     // Editor tab
     //////////////////////////////////////////////////////////////////////////////////////////////////////
     case WIDX_TITLE_EDITOR_ADD_SAVE:
-        if (!_isSequenceReadOnly && !_isSequencePlaying && !commandEditorOpen) {
+        if (!_isSequenceReadOnly && !_isSequencePlaying && !commandEditorOpen)
+        {
             auto intent = Intent(WC_LOADSAVE);
             intent.putExtra(INTENT_EXTRA_LOADSAVE_TYPE, LOADSAVETYPE_LOAD | LOADSAVETYPE_GAME);
             intent.putExtra(INTENT_EXTRA_CALLBACK, (void *) window_title_editor_add_park_callback);
             context_open_intent(&intent);
         }
         break;
     case WIDX_TITLE_EDITOR_REMOVE_SAVE:
-        if (window_title_editor_check_can_edit()) {
-            if (w->selected_list_item != -1) {
+        if (window_title_editor_check_can_edit())
+        {
+            if (w->selected_list_item != -1)
+            {
                 TitleSequenceRemovePark(_editingTitleSequence, w->selected_list_item);
-                if (w->selected_list_item >= (sint16)_editingTitleSequence->NumSaves) {
+                if (w->selected_list_item >= (sint16)_editingTitleSequence->NumSaves)
+                {
                     w->selected_list_item--;
                 }
             }
         }
         break;
     case WIDX_TITLE_EDITOR_RENAME_SAVE:
-        if (window_title_editor_check_can_edit()) {
-            if (w->selected_list_item != -1) {
+        if (window_title_editor_check_can_edit())
+        {
+            if (w->selected_list_item != -1)
+            {
                 window_text_input_open(w, widgetIndex, STR_FILEBROWSER_RENAME_SAVE_TITLE, STR_TITLE_EDITOR_ENTER_NAME_FOR_SAVE, STR_STRING, (uintptr_t)_editingTitleSequence->Saves[w->selected_list_item], 52 - 1);
             }
         }
         break;
     case WIDX_TITLE_EDITOR_LOAD_SAVE:
-        if (w->selected_list_item >= 0 && w->selected_list_item < (sint16)_editingTitleSequence->NumSaves) {
+        if (w->selected_list_item >= 0 && w->selected_list_item < (sint16)_editingTitleSequence->NumSaves)
+        {
             TitleSequenceParkHandle * handle = TitleSequenceGetParkHandle(_editingTitleSequence, w->selected_list_item);
             const utf8 * extension = path_get_extension(handle->HintPath);
             bool isScenario = park_importer_extension_is_scenario(extension);
             park_importer_load_from_stream(handle->Stream, handle->HintPath);
-            if (isScenario) {
+            if (isScenario)
```

Comment:
You _can_ leave out braces on if-elses (like you did here), but then I prefer a blank line before and after to make reading easier.